mobile/src/home-view.js

```diff
@@ -279,16 +330,22 @@ class HomeView extends Component {
     if (questionName.length === 0) {
       this.setState({showError: "red"})
     }
+    var approveVar = true
```

Comment:
prefer `let` (or `const` when the value will not change) to `var`.

Reply:
updated